src/cdk/v2/destinations/koddi/data/ClicksConfig.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "sourceKeys": "properties.tracking_data",
+    "required": true,
+    "destKey": "trackingData"
+  },
+  {
+    "sourceKeys": "properties.rank",
+    "required": true,
+    "destKey": "rank"
+  },
+  {
+    "sourceKeys": "properties.beacon_issued",
+    "required": true,
+    "destKey": "beaconIssued"
+  },
+  {
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true,
+    "required": true,
+    "destKey": "userGuid"
+  },
+  {
+    "sourceKeys": "properties.test_version_override",
```

Comment:
is this static per account basis or per event it will be different?

Reply:
This field will be used for A/B testing. It can be different for different event.